### PR TITLE
Don't use is="" with custom-style

### PR DESCRIPTION
**Pull request by rictic**

### CODE REVIEW

File: src/html/custom-style-no-is.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+  upgrade(document: ParsedHtmlDocument) {
+    const isBadCustomStyle =
+        p.AND(p.hasTagName('style'), p.hasAttrValue('is', 'custom-style'));
+    for (const style of dom5.nodeWalkAll(document.ast, isBadCustomStyle)) {
+      const parent = style.parentNode!;
+      const customStyle =
+          parse5.treeAdapters.default.createElement('custom-style', '', []);
+      dom5.insertBefore(parent, style, customStyle);
+      dom5.remove(style);
```
Comment (justinfagnani) [inline src/html/custom-style-no-is.ts]: remove shouldn't be necessary if you only append via dom5

File: src/html/custom-style-no-is.ts [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+      dom5.removeAttribute(style, 'is');
+      const include = dom5.getAttribute(style, 'include');
+      if (include != null) {
+        dom5.setAttribute(customStyle, 'include', include);
+        dom5.removeAttribute(style, 'include');
+      }
+      customStyle.childNodes = customStyle.childNodes || [];
```
Comment (justinfagnani) [inline src/html/custom-style-no-is.ts]: `append()` should work with no childNodes with dom5 2.0.0, and it'll correctly remove the element from existing parents.

**Review by justinfagnani — APPROVED** [review]

File: src/html/custom-style-no-is.ts (hunk 1 above)
Comment (rictic) [inline src/html/custom-style-no-is.ts]: Oh, nice! Done

File: src/html/custom-style-no-is.ts (hunk 2 above)
Comment (rictic) [inline src/html/custom-style-no-is.ts]: Done.

**Pull request merged**